public/scss/_base.scss

```diff
@@ -76,6 +76,62 @@ li:not(:last-of-type) {
   @extend %page-container;
 }
 
+.fip-container {
```

Comment:
Actually, can you make this an extra file? We should try to keep base as small as reasonable.